pom.xml

```diff
@@ -472,8 +472,7 @@
         <module>ambari-funtest</module>
         <module>ambari-agent</module>
         <module>mpack-instance-manager</module>
-        <module>ambari-client</module>
-        <module>ambari-shell</module>
+        
```

Comment:
`ambari-client` and `ambari-shell` modules were removed in trunk.  Should be removed from all profiles.

Reply:
Fixed.